src/main/scala/net/snowflake/spark/snowflake/io/CloudStorageOperations.scala

```diff
@@ -52,7 +54,8 @@ import scala.collection.JavaConversions._
 
 object CloudStorageOperations {
   private[io] final val DEFAULT_PARALLELISM = 10
-  private[io] final val S3_MAX_RETRIES = 3
+  private[io] final val S3_MAX_RETRIES = 10
```

Comment:
10 retries, with the sleep time doubled every time. That means we can end up sleeping for 2000 seconds in the last retry. That means if anyone has this mode turned on and they have a more simple error, they can wait a long time before they find out. Should this be the default value? Also, should this be configurable?

Reply:
The initial backoff is 100ms not 1s, so I think the last sleep should be 2**9 * 0.1 = 50s. Currently, it is not configurable. The reason is that we have introduced retry on spark connector which is configurable. It sounds no necessary to introduce two max retry count.  I admit it should be better to use smaller API retry count. I will change the API max retry count as 6. 2**6 * 0.1 = 6.4s. What do you think.